### PR TITLE
Remove elses

### PR DESCRIPTION
**Pull request by omarkdev**
Refactor some conditions removing else.When we have 'else' we open the possibility to add more steps and increase the cyclomatic complexity

| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no 
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php [hunk 1]
```diff
@@ -94,10 +94,9 @@ public function handleRequest(FormInterface $form, $request = null)
                 return;
             }
 
+            $data = $params ?: $files;
```
Comment (Tobion) [inline src/Symfony/Component/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php]: that creates an additional assignment when overwritten below

**Comment by Tobion** [comment]
Should be based on 3.4 to avoid conflicts later.

**Comment by omarkdev** [comment]
@Tobion so I need to apply this changes on 3.4 branch?

**Comment by nicolas-grekas** [comment]
Sorry for closing this right away, but similar changes have been submitted several times and have always been refused.
The reason is that this is pure CS change that creates maintainance burden and/or can introduce subtle bugs (happened before).
Thanks for proposing.

**Pull request closed without merging**